markdown/events/2021/bytesize-10-institutional-profiles.md

```diff
@@ -41,3 +41,250 @@ You can see the slides on HackMD: <https://hackmd.io/@jfy133/Hyc_WvXU_> (shown b
+
+[9:35](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=575) so once you have all this information, you can get started with writing.
+
+[9:40](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=580) We recommend starting with making a fork of the `nf-core/configs` repository.
+
+You don’t have to do this, but it generally makes life easier when you want to basically image everything.
+
+You can make a branch to follow GitHub best practices and just name it after the cluster name you’ve picked.
+
+Then also open a tab with an extra documentation because you will have to refer to it quite a lot.
+
+[10:03](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=603) Once you’ve made your fork and your branch, you need to create two new files.
+
+One under the `configuration` directory, which is where you’re going to put in the configuration file itself, a documentation file in markdown, and docs.
+
+You then need to add your profile name to these three files that nf-core uses to organise its stuff.
+
+It’s quite straightforward where to put your name there, just look in and you will find where the long lists of the other profiles are.
+
+[10:32](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=631) Now to write the profile itself. So in the configuration file, you can start by writing the `param scope`, this is basically standard Nextflow parameters, which you often specify in the command line.
+
+[10:44](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=644) So for nf-core profiles, we generally suggest that you add these config profile description content in url. This is useful later on for debugging, and it also makes it easier for users to know that they’re using the right profile when they are running their parameters.
+
+[10:59](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=659) Then in the `max` parameters that you see in the 4th to the 6th line, you can specify the maximum and node information that you gathered earlier. So the maximum number makes for memory that’s available, number of CPUs, and so on.
+
+[11:14](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=674) If you for example have a common directory for things like reference files, which you use in the network pipelines themselves, such as the Illumina iGenomes resource, you can also specify this here.
+
+[11:30](https://youtu.be/Ym1s6sKGzkw?list=PL3xpfTVZLcNiSvvPWORbO32S1WDJqKp1e&t=690) Once you specify that, you can then write the process scope.
+
+So this is actually where you define generic information about your particular scheduler.
+
+So in the example above (see Simple Example on the slide), we see that some clusters can have, and you just specify using slurm and maybe add an additional security check so that you can only retry a maximum of two times to stop runaway processes before you hit the big resource nodes.
```

Comment:
```suggestion
So in the example above (see Simple Example on the slide), we see the setup that some clusters can have, and you just specify using slurm and maybe add an additional security check so that you can only retry a maximum of two times to stop runaway processes before you hit the big resource nodes.
```